src/main/webapp/css/upload.css

```diff
@@ -18,7 +18,7 @@ input {
 }
 
 #upload-form {
-  background: linear-gradient(#e3ffe7 0%, #d9e7ff 100%);
+  background: #f7f7f7;
```

Comment:
BTW, are we using this rounded background outline across the other pages as well?

Reply:
Added the outline background from the receipt analysis page.